packages/api/src/praise/controllers.ts

```diff
@@ -201,3 +202,49 @@ export const quantify = async (
   const response = await praiseDocumentListTransformer(affectedPraises);
   res.status(200).json(response);
 };
+
+export const quantifyMultiple = async (
```

Comment:
This function will need to have same logic used in the `quantify` function to determine *all* the praises with a `scoreRealized` affected by the changes. It probably makes sense to pull that logic out into a utility function and use it in both controllers.

The reason for that logic is so the "Duplicate Score" display is updated when the original praise's score is updated. See the screencast below:

https://user-images.githubusercontent.com/159270/175825820-fac4dde2-e2bc-4d8f-8bcf-2441e9875781.mp4